Docker/test/curl/test_curl.sh

```diff
@@ -20,6 +20,12 @@ check_if_up() {
 
 check_if_up "$WIKIBASE_SERVER" "/wiki/Main_Page"
 
-check_if_up "$WDQS_SERVER" "/bigdata/namespace/wdq/sparql"
+if [ -z ${SKIP_WDQS+x} ]; then
```

Comment:
Ugh, interesting… this is essentially an isset check? Why not just check for emptiness?